proposals/forms.py

```diff
@@ -616,35 +616,11 @@ def clean(self):
             for study in self.instance.study_set.all():
                 documents = Documents.objects.get(study=study)
 
-                if study.passive_consent:
-                    if not documents.director_consent_declaration:
-                        self.add_error('comments', _(
-                            'Toestemmingsverklaring voor traject {} nog niet toegevoegd.').format(
-                            study.order))
-                    if not documents.director_consent_information:
-                        self.add_error('comments', _(
-                            'Informatiebrief voor traject {} nog niet toegevoegd.').format(
-                            study.order))
-                    if not documents.parents_information:
-                        self.add_error('comments', _(
-                            'Informatiebrief voor traject {} nog niet toegevoegd.').format(
-                            study.order))
-                else:
-                    if not documents.informed_consent:
-                        self.add_error('comments', _(
-                            'Toestemmingsverklaring voor traject {} nog niet toegevoegd.').format(
-                            study.order))
-                    if not documents.briefing:
-                        self.add_error('comments', _(
-                            'Informatiebrief voor traject {} nog niet toegevoegd.').format(
-                            study.order))
-
-                if study.needs_additional_external_forms():
-                    if not documents.director_consent_declaration:
-                        self.add_error('comments', _(
-                            'Toestemmingsverklaring voor traject {} nog niet toegevoegd.').format(
-                            study.order))
-                    if not documents.director_consent_information:
-                        self.add_error('comments', _(
-                            'Informatiebrief voor traject {} nog niet toegevoegd.').format(
-                            study.order))
```

Comment:
I think we still need this code? Desiree's last comment seems to suggest there forms are still required in some cases?

Reply:
Sadly that comment goes a bit beyond the scope of this PR. Once algemeen belang has been sorted out we can always make them required again.

Reply:
Fair enough, let's do that in a different PR then if necessary